src/components/a-form-input/_template.js

```diff
@@ -0,0 +1,52 @@
+import bel from 'bel';
+import classnames from 'classnames';
+
+export default function ({
```

Comment:
Hmh, I think it would be cool to support HTML5 input attributes as well.
May the easiest would be through rest `...args` or in this case `...props`.
I'm curious if **bel** understand object spread for attributes 🤔